airflow/www/templates/airflow/dags.html

```diff
@@ -119,12 +119,6 @@
       <a href={{ get_docs_url("executor/index.html") }}  target="_blank" rel="noopener noreferrer"><b>Click here</b></a> for more information.
     {% endcall %}
   {% endif %}
-  {% if otel_metrics_on | default(false) %}
```

Comment:
Add conversation about newsfragment here so that it's not merged accidentally.

Reply:
Dev list email sent this morning.  :+1: